### PR TITLE
refactor: remove highlightToggle

### PR DESCRIPTION
**Pull request by MarcelGerber**
[As discussed here](https://owid.slack.com/archives/CQQUA2C2U/p1643054986036100)

we haven't used it in any chart since 2020, so let's get rid of the code behind it now

### CODE REVIEW

**Review by danielgavrilov — APPROVED** [review]
🚀 

I just double checked, no live chart has this enabled. We removed it from all charts in 2020, because it was broken: [The custom 'highlight toggle' on ScatterPlot no longer works](https://www.notion.so/owid/The-custom-highlight-toggle-on-ScatterPlot-no-longer-works-eced49cb6d9f4322999fc32bb92780e2)

**Review by danyx23 — APPROVED** [review]
LGTM - I'll merge the schema branch today, if you merge this before that lands then I'll remove the highlightToggle field from the schema definition

**Pull request merged**
